src/CoreBundle/Factory/BackendTreeNodeFactory.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace ChameleonSystem\CoreBundle\Factory;
+
+use ChameleonSystem\CoreBundle\DataModel\BackendTreeNodeDataModel;
+
+class BackendTreeNodeFactory
+{
+    public function createTreeNodeDataModelFromTreeRecord(\TdbCmsTree $treeNode): BackendTreeNodeDataModel
+    {
+        $treeNodeDataModel = new BackendTreeNodeDataModel($treeNode->id, $treeNode->fieldName, $treeNode->sqlData['cmsident'], $this->getConnectedPageId($treeNode));
+        return $treeNodeDataModel;
```

Comment:
missing empty line before return (csfixer)

Reply:
No need of a variable here. Return the dataModel directly.

Reply:
ok